imports.go

```diff
@@ -131,6 +130,7 @@ func tryPath(dir, importedPath string) (found bool, content []byte, foundHere st
 func (importer *FileImporter) Import(dir, importedPath string) *ImportedData {
 	found, content, foundHere, err := tryPath(dir, importedPath)
 	if err != nil {
+		fmt.Println(err)
```

Comment:
Looks like abandoned debug print

Reply:
Right. Moreover the branch wasn't covered by tests so I didn't notice. 

Println removed, test added.